### ISSUE TITLE
lt-cred-mech deprecated?

### ISSUE THREAD

**Issue by lonesomewalker** [coturn/coturn#646]
As seen somewhere:
is the parameter lt-cred-mech deprecated since version 4.x...?

Can't find information to this.

Thanks.

**Comment by lonesomewalker** [comment]
FYI, here described:
https://nextcloud-talk.readthedocs.io/en/turn_doc/TURN/

**Comment by misi** [comment]
I don't think it is deprecated. It is outdated and not as secure as shared secret..
